### DISCUSSION TITLE
Back button not available in map and poi download

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#388] [Q&A] [answered]
just a little thing: the back button for staying in the menu (and not leaving the menu when pressing the Android back button) is not available in these submenus. Can you please also add it there, also in the Graphhopper download. Only when using the settings it is ok to leave the menu. Or is it better to also have the back symbol there?

**Comment by devemux86** [comment]
@DolomitesTom thanks for the suggestion.

Do you mean like the left arrow in the image?

Add a similar arrow in all download manager screens that will return to the main download menu?
(with the menu Map, POI, BRouter, GraphHopper)

<img src="https://github.com/devemux86/cruiser/assets/3484020/6ebdea88-8b91-40df-a02d-195e12e1bb5e" width="50%"/>

**Reply by DolomitesTom** [reply]
Yes exactly

**Comment by devemux86** [comment]
Similar back buttons are also needed in:
- Profile manager
- Subscriptions

**Comment by devemux86** [comment]
All back buttons will be available in the next Cruiser update.

**Comment by devemux86** [comment, marked as answer]
Implemented in [Cruiser 3.0.37](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
